### DISCUSSION TITLE
[Fastlane.swift] Getting Started

### DISCUSSION THREAD

**Discussion by leszko11** [fastlane/fastlane#22008] [Q&A]
Hey all 👋 

I'm trying to setup a Swift Package with `fastlane` added with SPM. I've followed [Get Started (SPM) (Beta)](https://docs.fastlane.tools/getting-started/ios/fastlane-swift/), but unfortunately I'm not able to have it running correctly. 

What I did:
✔️ create new executable project
✔️ add `Fastlane` as dependency
✔️ create `Fastfile.swift` file and added custom lane to it
✔️ start fastlane runloop
✔️ add arguments to target
❌ my custom lane caanout be found

When I inspect `Fastfile.swift` file included in fetched dependency I see this annotation
```
// This class is automatically included in FastlaneRunner during build
// If you have a custom Fastfile.swift, this file will be replaced by it
// Don't modify this file unless you are familiar with how fastlane's swift code generation works
// *** This file will be overwritten or replaced during build time ***
```

So this file should be overwritten with my `Fastfile.swift` content, right?

**Comment by mollyIV** [comment]
Hey @leszko11 , 

I double-checked the documentation and noticed a potential typo. 

Step 3 instructs you to create `Fastfile.swift` file and the class inside is named `FastFile`. 

```swift
import Fastlane

// Create a class with: 
class FastFile: LaneFile {
    // Your lanes goes here.
}
```

However, in Step 4, it says to run the main with `Fastfile`, which doesn’t match the class created in Step 3 — `FastFile`.

Could you please try running it with your FastFile instead?

```swift
import Fastlane

Main().run(with: FastFile()) // It's FastFile, not Fastfile class 
```
